CMakeLists.txt

```diff
@@ -65,6 +68,17 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
     endif()
 endif()
 
+add_library(D3D12Core SHARED IMPORTED)
+set_target_properties(D3D12Core PROPERTIES
+    IMPORTED_LOCATION ${GFX_AGILITY_PATH}/lib/D3D12Core.dll
+    IMPORTED_IMPLIB ${GFX_AGILITY_PATH}/lib/dummy.lib
+)
+add_library(d3d12SDKLayers SHARED IMPORTED)
+set_target_properties(d3d12SDKLayers PROPERTIES
+    IMPORTED_LOCATION ${GFX_AGILITY_PATH}/lib/d3d12SDKLayers.dll
+    IMPORTED_IMPLIB ${GFX_AGILITY_PATH}/lib/dummy.lib
+)
```

Comment:
Added the agility SK libs as cmake dependencies, this allows for importing projects to correctly install them as needed